package.json

```diff
@@ -16,6 +16,7 @@
     "url": "git+https://github.com/interledgerjs/ilp-plugin-btp.git"
   },
   "scripts": {
+    "prettier": "prettier --write 'src/*.ts' 'src/*/*.ts' 'test/*/*.js' 'test/*.js'",
```

Comment:
try double star globs, e.g. `src/**/*.ts` 

you can look at the coil-extension globs for example:
```
    "prettier": "prettier --write '*.{ts,tsx,js,html,jsx,md}' '{src,test}/**/*.{ts,tsx,js,html,jsx,md}'",
```